### PR TITLE
LPS-179960 Export DynamicObjectDefinitionTable and DynamicObjectRelationshipMappingTable

### PR DESCRIPTION
**Pull request by joseabelenda**
- LPS-179960 Move DynamicObjectDefinitionTable and DynamicObjectRelationshipMappingTable from service to api
- LPS-179960 Sort
- LPS-179960 Export com.liferay.object.dsl package
- LPS-179960 Auto SF

### CODE REVIEW

**Comment by joseabelenda** [comment]
Hey team!
To run DSLQueries with Objects is necessary to use Export DynamicObjectDefinitionTable and DynamicObjectRelationshipMappingTable, but both are internal classes.
To be possible use them outside object-service, I moved them to object-api and export them.
Please take a look on this PR and let me know if it makes sense or if there is any change request.
Thank you!

**Comment by joseabelenda** [comment]
@gabrielwas

File: modules/apps/object/object-api/bnd.bnd [hunk 1]
```diff
@@ -10,6 +10,7 @@ Export-Package:\
 	com.liferay.object.definition.notification.term.util,\
 	com.liferay.object.deployer,\
 	com.liferay.object.display.context,\
+	com.liferay.object.dsl,\
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-api/bnd.bnd:13]: @joseabelenda 

Does use `com.liferay.object.petra.sql.dsl,\` to keep the same path used in **object-service** make sense for you?

I mean, is there a reason why you opted to define as `com.liferay.object.dsl`

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hi @joseabelenda 

Your changes looks good to me, I've left only one question

Thanks for you changes!
Feliphe

File: modules/apps/object/object-api/bnd.bnd (hunk 1 above)
Comment (joseabelenda) [reply modules/apps/object/object-api/bnd.bnd]: Ok. It makes sense.
Thank you.

**Comment by joseabelenda** [comment]
ci:test:relevant

**Comment by joseabelenda** [comment]
ci:test:object

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by gabrielwas** [comment]
ci:forward

**Pull request closed without merging**